messages/recordcounts.json

```diff
@@ -0,0 +1,8 @@
+{
+  "commandDescription": "print record counts for the given objects",
+  "examples": [
+    "sfdx force:limits:recordcounts:display -o Account,Contact,Lead,Opportunity",
```

Comment:
As per our naming guidelines, we suggest using "-s" short name for object types.  BUT... we use -s for a bunch of other things too, so honestly I think -o is fine.  But it's my job to point this out and see what others think :)

https://developer.salesforce.com/docs/atlas.en-us.sfdx_cli_plugins.meta/sfdx_cli_plugins/cli_plugins_style_parameter_names.htm

Reply:
I think... I'll follow the naming guidelines :)